doc/source/whatsnew/v0.21.0.txt

```diff
@@ -53,6 +53,7 @@ Backwards incompatible API changes
 - :func:`read_csv` now treats ``'n/a'`` strings as missing values by default (:issue:`16078`)
 - :class:`pandas.HDFStore`'s string representation is now faster and less detailed. For the previous behavior, use ``pandas.HDFStore.info()``. (:issue:`16503`).
 - Compression defaults in HDF stores now follow pytable standards. Default is no compression and if ``complib`` is missing and ``complevel`` > 0 ``zlib`` is used (:issue:`15943`)
+- Index.get_indexer_non_unique() now returns a ndarray indexer rather than an Index; this is consistent with Index.get_indexer() (:issue:`16819`)
```

Comment:
use double-backticks on ``Index.get_indexer_non_unique()``, ``Index``, and ``Index.get_indexer()``